agent/docker/jvm/Dockerfile.alpine

```
@@ -5,9 +5,10 @@ RUN go get -d -v ./...
 RUN cd agent && go build -o /go/bin/agent
 
 FROM openjdk:8-alpine as asyncprofiler
-RUN apk add build-base git linux-headers
-RUN git clone https://github.com/edeNFed/async-profiler.git
-RUN cd async-profiler && make
+RUN apk add build-base linux-headers curl
```

For unpacking the tar.gz file, the alpine tar package is needed because the existing `tar` command from busybox doesn't support all necessary features.

```suggestion
RUN apk add build-base linux-headers curl tar
```